.gitignore

```diff
@@ -2,3 +2,4 @@ _site/
 .sass-cache/
 oresat-design
 field-of-view
+*.swp
```

Comment:
This is fine, but FYI you should always add this to your global gitignore on your machine. Saves you from having to remember to add it to every single different gitignore in every single repo ever.

Reply:
Ah, yes, that is a nicer way to do it. :smile: